tests/MSBuildDeviceIntegration/MSBuildDeviceIntegration.csproj

```diff
@@ -17,6 +17,7 @@
     <Compile Include="..\..\src\Xamarin.Android.Build.Tasks\Tests\Xamarin.Android.Build.Tests\Utilities\*.cs">
       <Link>Utilities\%(FileName).cs</Link>
     </Compile>
+    <Compile Include="..\..\tools\xabuild\SymbolicLink.cs" />
```

Comment:
If we keep `BaseTest.cs` unchanged, you could remove this one, too. I missed it.